test-files/algorithms/changeset/ChangesetCreatorTest/ChangesetDeriveUpdateVersion_sec.osm

```diff
@@ -0,0 +1,235 @@
+    <node visible="true" id="49" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4765725000000032" lon="-71.4718222999999995"/>
+    <node visible="true" id="50" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4762861000000029" lon="-71.4726983000000047"/>
+    <node visible="true" id="51" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4763586999999987" lon="-71.4726952000000040"/>
+    <node visible="true" id="52" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4763519999999986" lon="-71.4724046000000044"/>
+    <node visible="true" id="53" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4762794000000028" lon="-71.4724075999999968"/>
+    <node visible="true" id="54" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4762873000000027" lon="-71.4727948999999967"/>
+    <node visible="true" id="55" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764081000000004" lon="-71.4727898999999951"/>
+    <node visible="true" id="56" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764063000000007" lon="-71.4728142000000020"/>
+    <node visible="true" id="57" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764297999999982" lon="-71.4728152999999935"/>
+    <node visible="true" id="58" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764333999999977" lon="-71.4728905999999995"/>
+    <node visible="true" id="63" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764411000000024" lon="-71.4714591000000041"/>
+    <node visible="true" id="64" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764478999999966" lon="-71.4721255000000042"/>
+    <node visible="true" id="65" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4764023999999978" lon="-71.4721322999999984"/>
+    <node visible="true" id="66" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4763260000000002" lon="-71.4721265000000017"/>
+    <node visible="true" id="67" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4762902000000011" lon="-71.4715712999999937"/>
+    <node visible="true" id="68" timestamp="1970-01-01T00:00:00Z" version="1" lat="42.4762970000000024" lon="-71.4714532999999932"/>
+    <way visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1">
+        <nd ref="-1"/>
+        <nd ref="-2"/>
+        <nd ref="-3"/>
+        <nd ref="-4"/>
+        <nd ref="-1"/>
+        <tag k="feature_area" v="112.5"/>
+        <tag k="name" v="Twin Seafood"/>
+        <tag k="building" v="yes"/>
+        <tag k="length" v="13.6"/>
+        <tag k="amenity" v="cafe"/>
+        <tag k="width" v="8.5"/>
+        <tag k="angle" v="179.5"/>
+        <tag k="addr:housenumber" v="541"/>
+        <tag k="addr:street" v="Massachusetts Avenue"/>
+        <tag k="addr:state" v="MA"/>
+        <tag k="addr:country" v="US"/>
+        <tag k="addr:city" v="Acton"/>
+    </way>
+    <way visible="true" id="1" timestamp="1970-01-01T00:00:00Z" version="2">
```

Comment:
Ways 1, 6, 8, and 9 have all had their versions incremented in the secondary dataset as the result of a join with a POI (done by hand).